funnel/views/schedule.py

```diff
@@ -185,15 +201,22 @@ def subscribe_schedule(self):
     @cors('*')
     @requires_permission('view')
     def schedule_json(self):
-        return jsonp(schedule=schedule_data(self.obj),
+        return jsonp(
+            schedule=schedule_data(self.obj),
             venues=[venue.current_access() for venue in self.obj.venues],
-            rooms=[room_data(room) for room in self.obj.rooms])
+            rooms=[room_data(room) for room in self.obj.rooms],
+        )
 
     @route('ical')
     @requires_permission('view')
     def schedule_ical(self):
         cal = Calendar()
-        cal.add('prodid', "-//Schedule for {event}//funnel.hasgeek.com//".format(event=self.obj.title))
+        cal.add(
+            'prodid',
+            "-//Schedule for {event}//funnel.hasgeek.com//".format(
```

Comment:
this url needs to be updated.

Reply:
We're not doing non-formatting related fixes in this PR.